test/unit/util/webhooks.test.ts

```diff
@@ -0,0 +1,87 @@
+import { calculateProcessingTimeInSeconds } from "../../../src/util/webhooks";
+
+let dateNowSpy;
+
+const currentTime = 1632726221873;
+
+beforeAll(() => {
+	// Lock Time
+	dateNowSpy = jest.spyOn(Date, "now").mockImplementation(() => currentTime);
+});
+
+afterAll(() => {
+	// Unlock Time
+	dateNowSpy.mockRestore();
+});
+
+describe("Webhooks suite", () => {
+	const mockInfoLog = jest.fn();
+	const mockContextLogger = {
+		info: mockInfoLog,
+	};
+
+	describe("calculateProcessingTimeInSeconds", () => {
+		it("should calculate processing time for webhook events", () => {
+			const webhookReceived = 1632726220873;
+			const webhookName = "workflow_run";
+			const status = 202;
+			const result = currentTime - webhookReceived; // 1000ms
```

Comment:
/nit for tests it would be better to use simpler times like currentTime=2000 webhookReceived=500 and the expect the result be 500 (rather than redo calculations of the function within the test)